prov/rxm/src/rxm_init.c

```diff
@@ -106,6 +108,11 @@ int rxm_info_to_core(uint32_t version, const struct fi_info *hints,
 		}
 	}
 	core_info->ep_attr->type = FI_EP_MSG;
+	if (fi_param_get_bool(&rxm_prov, "use_srx", &use_srx) && use_srx) {
```

Comment:
This check should be: if (!fi_param_get_bool(....). With that change RxM correctly selects the XRC info if FI_OFI_RXM_USE_SRX=1 is specified and and the XRC info is prioritized (FI_VERBS_PREFER_XRC=1) or the "-xrc" domain is specified.

Reply:
Oops! It's fixed now. Thanks for verifying!